CWE-89/doctrine-dbal-dangerous-query.php

```
@@ -0,0 +1,57 @@
+        $conn = $this->getEntityManager()->getConnection();
+        $sql = "SELECT * FROM users WHERE username = ?";
+// {fact rule=sql-injection@v1.0 defects=0}
+        // ok: doctrine-dbal-dangerous-query
+        $stmt = $conn->prepare($sql);
+// {/fact}
+        $stmt->bindValue(1, $_GET['username']);
+        $resultSet = $stmt->executeQuery();
+        return $resultSet;
+    }
+
+    public function okTest2(int $price): array
```

PHPMD (2.15.0)

Avoid unused parameters such as '$price'. (undefined)

(UnusedFormalParameter)